content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+In this case, for management operations, such as creating accounts, IAM accounts, buckets, etc., I recommend you use the `mc` command line tool.
+
+For object listing, getting, putting, and deleting operations, I recommend you use `AWS CLI`.
+
+#### 3. Cannot revert to HTTP mode after enabling HTTPS.
+
+If the Object Store was installed using the Installer, the Object Store would also have security enabled, including HTTPS.
+
+Although you can see the below description:
+
+> To revert to http access, comment out the moss.certs.dir=/opt/mapr/conf line in the /opt/mapr/conf/moss.conf file.
+
+from here - [HTTP Access to Object Store](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html#concept_isb_53h_5bb__section_bg1_zd1_vsb), but this configuration change alone does not change Object Store to HTTP mode, and there is nothing else in HPE Ezmeral Data Fabric's documentation on how to modify Object Store's TLS mode.
+
+### [](#create-a-bucket-in-hpe-ezmeral-data-fabric-object-store-and-upload-some-objects)Create a bucket in HPE Ezmeral Data Fabric Object Store and upload some objects
+
+In order to create a Bucket, you need to create an Account, an IAM User, and finally a Bucket in sequence.
+While creating IAM User and Bucket, you also need to prepare Access Policy to control who (IAM User) can perform which (list, get, put, etc.) operations on which Bucket.
+
+You can read the following document - [Entities and Resources](https://docs.datafabric.hpe.com/72/MapROverview/object_store_account_management.html) first to gain a deeper understanding of the entity model of HPE Ezmeral Data Fabric Object Store.
+
+Below, I will demonstrate in turn how to create the above-required entities.
+
+#### Create an account
+
+I choose to use the [mc](https://docs.datafabric.hpe.com/72/ReferenceGuide/mc-commands-overview.html) command line tool to create Account.
+
+Before using the mc command line for the first time, you need to create an Alias for your administrator.
+
+An Alias contains an access endpoint, such as "https://s3-us-west-1.amazonaws.com", which is an Amazon AWS S3 endpoint; another example is "http://10.10.88.198:9000", which is a Minio endpoint.
+An Alias also contains the Access Key and Secret Key used by your administrator or IAM User.
+
+##### 1. You first use the "[mc alias list](https://docs.datafabric.hpe.com/72/ReferenceGuide/mc-alias-list.html)" command to view the default Alias in the following systems.
+
+❗Note: If you are using self-signed TLS certificates or installed the cluster by Installer, you have to copy <ins>/opt/mapr/conf/ca/chain-ca.pem</ins> to <ins>~/.mc/certs/CAs/</ins> on the node running `mc`.
```

Comment:
Modify "...installed the cluster by Installer" to "...installed the cluster via Installer"

Modify "The meaning of this step is the same as you imported the self-issued CA..." to "The reason for this step is the same as why you imported the self-issued CA..."